content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,178 @@
+  <https://trendnet.com/products/proddetail.asp?prod=265_TU-S9>`__); and
+- An :ref:`IoT Box <belgium/iotbox>` (one IoT box per black box);
+- A receipt printer.
+
+.. _belgium/blackbox:
+
+Black box
+*********
+
+As a pre-requisite, activate the **black box module**. To do so, go to the :guilabel:`Apps`
+application, remove the :guilabel:`Apps` filter, search for the `Belgian Registered Cash Register`
+module (technical name: `pos_blackbox_be`), and click :guilabel:`Activate`.
+
+.. image:: belgium/be-modules.png
+   :align: center
+   :alt: black box modules for belgian fiscal certification
+
+Once the module is activated, update your company info with your VAT number. To set it up, go to
+:menuselection:`Settings --> Companies --> Update Info`, and fill in the :guilabel:`VAT` field.
+Then, enter a national registration number for every staff member who operates the POS system. To do
+so, go to the :guilabel:`Employees` app and open an employee form. After that, go to the
+:menuselection:`HR settings tab --> Attendance`, and fill in the :guilabel:`INSZ or BIS number`
+field.
+
+.. image:: belgium/bis-number.png
+   :align: center
+   :alt: ISNZ or BIS number field on employee form
+
+.. tip::
+   To input your information, click on your avatar, go to :menuselection:`My Profile --> Preference
+   tab`, and enter your INSZ or BIS number in the designated field.
+
+.. warning::
+   Configuring the black box directly in the production database is essential, as utilizing it in a
+   testing environment may result in incorrect data being stored within the black box.
+
+.. _belgium/iotbox:
+
+IoT Box
+*******
+
+In order to use a Fiscal Data Module, you need a registered IoT Box. To register your IoT box, you
+must contact us through our `contact form <https://www.odoo.com/help>`_ and provide the following
+information:
+
+- your VAT number;
+- your company's name, address, and legal structure; and
+- the Mac address of your IoT Box.
+
+Once your IoT box is certified, :doc:`connect <../../productivity/iot/config/connect>` it to your
+database. To verify that the IoT Box recognizes the FDM, go to the IoT homepage and scroll down the
+:guilabel:`IOT Device` section, which should display the FDM.
+
+.. image:: belgium/iot-devices.png
+   :align: center
+   :alt: Hardware status page on a registered IoT Box
+
+Then, add the IoT to your POS. To do so, go to :menuselection:`Point of Sale --> Configuration -->
+Point of Sale`, select your POS, scroll down to the :guilabel:`Connected Device` section, and enable
+:guilabel:`IoT Box`. Lastly, add the FMD in the :guilabel:`Fiscal Data Module` field.
+
+.. note::
+   To be able to use an FDM, you must at least connect a :guilabel:`Receipt Printer`.
+
+.. _belgium/vat:
+
+VAT signing card
+----------------
+
+When you open a POS session and make your initial transaction, you are prompted to enter the PIN
+provided with your :abbr:`VSC (VAT signing card)`. The card is delivered by the :abbr:`FPS (Service
```

Comment:
We could never trigger this again, but it doesn't seem to be useful. (the pin code)